Corrected outdated Information and added website explanation (#12253)
ppy/osu-wiki: wiki/Community/Project_Loved/en.md

- [osu!catch submissions](https://loved.sh/submissions/fruits)
- [osu!mania submissions](https://loved.sh/submissions/mania)

> Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category. If a beatmap fails the voting they are eligible for a second shot at Loved.

Reviewer: seems misleading as it implies that a failed beatmap is guaranteed a second shot, which is not the case.

something like this could work better:
```suggestion
Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category. A failed vote does not mean a beatmap would lose its chance to be re-nominated in the future.
```

Author: But doesn't your suggestion leave the fact out that it can only have a second chance. Yours sounds like it can fail voting without a limit of attempts

- At least one difficulty can be passed
- Every nominated difficulty has a [drain time](/wiki/Beatmap/Drain_time) of at least 20 seconds

> Although many unranked beatmaps meet these criteria, few are voted into the Loved category due to the selective nature of the captains' picks, and the high community approval requirement by voting.

Reviewer: why remove this? It's still accurate to the point it feels like a warranted disclaimer.

Author: Thought it was unnecessary since it's obvious that not ever map can have it's turn, but if you rly want to I will add it back in there.

Author: ok

In some cases, beatmaps may be nominated despite breaking some of the criteria, as long as the mapper can update it to abide by the criteria prior to Loved. For example, a beatmap with an inappropriate background may be nominated as long as the background is later removed or replaced. An active contributor to Project Loved will usually post on the beatmap's [discussion](/wiki/Beatmap_discussion) if any changes are required.

Additionally, contributors to Project Loved often make technical suggestions, such as improvements to audio quality, [timing](/wiki/Beatmapping/Timing) accuracy, or [metadata standardisation as recommended by the ranking criteria](/wiki/Ranking_criteria/Metadata). The mapper is strongly encouraged to apply these suggestions, but they are not required to.

> ### osu!standard-specific criteria

Reviewer: osu!standard is no longer officially-supported terminology, you just call it `osu!`.

Author: Shall do

| Category | Explanation |
| :-: | :-- |
| Popular Playcounts 1 | Popular beatmaps in the community, with at least 100,000 playcount. |

> | Popular Playcounts 2 | Popular beatmaps in the community, with at least 100,000 playcount. |

Reviewer: mentioning this twice feels redundant imo.
```suggestion
```

Author: Woudln't 2 Popular Playcounts be better to imply that it's 2 maps for that category.
Popular Playcount 2 sounds like it describes a different ruleset as Popular Playcount 1 (it's obviously the same but it sounds that way)

**Guidelines may be ignored under exceptional circumstances.** These exceptional circumstances can be justified by an explanation as of why the guideline has been ignored and why not ignoring it will interfere with the overall quality of the creation.

> - **The beatmap must be submitted at least 1 year before Loved nomination, except for beatmaps that break ranking criteria due to their concept.** Submitting a beatmap solely for Loved is discouraged when Ranked might be a better option. Exceptions include sliderator usage, Aspire maps, storyboard gameplay gimmicks, etc.

Reviewer: why is this being moved to guidelines? it's a hard rule that doesn't have much ambiguity.

if you're proposing changes to rules and guidelines, they should be passed on and discussed with the Loved team.

best to move this back to rules for now.

Author: Talked about this in the osu dev Discord and this is what was huu told me, so I moved it there

![Loved Guideline](https://github.com/user-attachments/assets/88a188fd-70de-45f3-be9b-1eb617984385)

Reviewer: in that case, it's recommended to use slightly less imperative wording for guidelines
```suggestion
- **The beatmap should be submitted at least 1 year before Loved nomination, except for beatmaps that break ranking criteria due to their concept.** Submitting a beatmap solely for Loved is discouraged when Ranked might be a better option. Exceptions include sliderator usage, Aspire maps, storyboard gameplay gimmicks, etc.
```

There are no strict rules governing which maps a captain must nominate for Loved, but generally the nominated maps are very well known and appreciated by the community (as the title "Loved" implies). Sometimes a beatmap will also be preferred if it is significantly older than others or contains beatmaps with unique and high-quality mapping styles.

## Website

> All of the submitted beatmaps can be seen at the [Loved website](https://loved.sh). 

Reviewer 2: This introductory sentence should be reworded entirely in my opinion.

There isn't any previous reference to "submitted beatmaps". The introduction to this new section should first and foremost introduce the site and then explain how it acts as the main submission hub for users to promote maps towards the Loved status.

Author: The website is mentioned in line 11 under "To submit a beatmap for Loved, please visit the [beatmap submission page](https://loved.sh/submit) of the Project Loved website and fill out the form."

Reviewer 2: Despite having a mention there, considering it was mentioned up to almost 100+ lines up and we're adding a new header for the Website alone, the introduction should be much meatier than that. Readers will expect to know that information just by going to its respective section.

Author: Aight, makes sense

## Website

> The [Loved website](https://loved.sh) serves as the hub for everything related to Project Loved. Here, you can view submitted beatmaps, submit your own, browse past votings, and much more.

Reviewer 2: ```suggestion
The [Loved website](https://loved.sh) serves as the hub for everything related to Project Loved. Here, you can view submitted beatmaps, submit your own, browse past votings, and much more. It is also used by the [Project Loved Team](/wiki/People/Project_Loved_Team) to handle several miscellaneous tasks when nominating maps.
```

Would specify that the site is used and managed by basically all - if not most - team members at this point.

Author: Did so
